Kernel/Arch/init.cpp

```diff
@@ -446,6 +446,8 @@ void init_stage2(void*)
     auto userspace_init = kernel_command_line().userspace_init();
     auto init_args = kernel_command_line().userspace_init_args();
 
+    dmesgln("Running first user process: {}", userspace_init);
+    dmesgln("Init (first) process args: {}", init_args);
```

Comment:
perhaps we should do a VFS lookup on this path to print its real name if it's a symlink?